src/native/http_request_response.c

```diff
@@ -116,17 +131,17 @@ int aws_java_http_stream_on_incoming_headers_fn(
     void *user_data) {
     (void)block_type;
 
-    struct http_stream_binding *callback = (struct http_stream_binding *)user_data;
+    struct http_stream_binding *binding = (struct http_stream_binding *)user_data;
     int resp_status = -1;
     int err_code = aws_http_stream_get_incoming_response_status(stream, &resp_status);
     if (err_code != AWS_OP_SUCCESS) {
         AWS_LOGF_ERROR(AWS_LS_HTTP_STREAM, "id=%p: Invalid Incoming Response Status", (void *)stream);
         return AWS_OP_ERR;
     }
 
-    callback->response_status = resp_status;
+    binding->response_status = resp_status;
```

Comment:
thank you for renaming all of these
I'm sure there's some historical reason these were all named `callback`, but that should have changed long ago